__tests__/pages/telekonseling.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+import { render, screen } from "@testing-library/react";
+import TelecounselingPage, { getStaticProps } from "~/pages/telekonseling";
+import telekonseling from "~/lib/content/telekonseling";
+
+jest.mock("~/lib/content/telekonseling");
+jest.mock("next/router", () => require("next-router-mock"));
+
+describe("TelecounselingPage", () => {
+  const { supports } = telekonseling;
+
+  it("renders the title, description and breadcrumbs correctly", async () => {
+    render(<TelecounselingPage telekonseling={telekonseling} />);
+
+    const title = screen.getByRole("heading", {
+      name: /Curhat atau Telekonseling/i,
+    });
+    expect(title).toBeVisible();
+
+    const description = screen.getByText(
+      /Informasi mengenai relawan COVID-19 yang memberikan bantuan berupa layanan konsultasi psikologi, kesehatan, agama, dan lainnya yang relevan untuk membantu kesehatan mental terdampak COVID-19.$/i,
+    );
+    expect(description).toBeVisible();
+
+    const breadcrumbs = screen.getByRole("link", {
+      name: "Curhat/Telekonseling",
+    });
+    // const breadcrumbs = screen.getByText(/^Curhat atau Telekonseling$/i);
+    expect(breadcrumbs).toBeVisible();
+    expect(breadcrumbs).toHaveAttribute("href", "/telekonseling");
+    // breadcrumbs.getAttribute("href") === "telekonseling";
```

Comment:
```suggestion
    expect(breadcrumbs).toBeVisible();
    expect(breadcrumbs).toHaveAttribute("href", "/telekonseling");
```

Please remove those commented lines.